HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -136,7 +136,11 @@ def self.add_zones_spaces_if_needed(hpxml, hpxml_bldg, cfa)
 
   def self.apply_header(hpxml_header, epw_file, hpxml_bldg)
     if hpxml_header.timestep.nil?
-      hpxml_header.timestep = 60
+      if (not hpxml_header.geb_onoff_thermostat_deadband.nil?) || (not hpxml_header.geb_backup_heating_capacity_increment.nil?)
+        hpxml_header.timestep = 1
```

Comment:
I'm a bit on the fence as to whether we should do this. If we do, it needs to be documented. (Are there other cases, like the advanced defrost model, where we should also default to 1?)

But I lean towards not doing this. Rather, we should make someone specify a timestep of 1 so they know what they're getting themselves into when they choose one of these advanced research features.